MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflMainViewPresenter.h

```diff
@@ -53,6 +53,7 @@ class MANTIDQT_CUSTOMINTERFACES_DLL ReflMainViewPresenter
                             boost::shared_ptr<IReflSearcher>());
```

Comment:
This thing should inherit from `public IReflOuterPresenter`. Multiple inheritance of pure-virtual interfaces is not a problem.